### ISSUE TITLE
Day time moves backwards when server is frozen using Slumber

### ISSUE THREAD

**Issue by ViciousBadger** [wendall911/BetterDays#18]
Hi, there is a strange behavior when using Slumber and BetterDays on a server. I have my day and night speed both set to 0.3333 for one hour day cycles, but I also use Slumber to pause the server when no-one is online. The issue is that time seems to be moving backwards when the server is frozen. I checked with "time query daytime" repeatedly and the number keeps getting smaller, even going below zero eventually.

I tested this with only Slumber, BetterDays and Fabric API to make sure no other mods are interfering and the issue is still present.

**Comment by wendall911** [comment]
Yeah, they do some not great things to "pause" server. We tried lots of them until we made our own a few years back. I don't know that I can make do compat, but will take a look. Here is our server pause utility mod: https://github.com/wendall911/ReadyPlayerFun which works with any day/night cycle mod.

I'll take a look and see what the issue is with Slumber this coming week, as I'll be out of town for a few days.

**Comment by ViciousBadger** [comment]
Hey that's pretty cool, hadn't thought about using that mod. Do you know if it also decreases server load when paused? (That's the reason I use slumber)

**Comment by nx5314** [comment]
This seems to be happening with the new vanilla `pause-when-empty-seconds` feature as well

**Comment by wendall911** [comment]
@nx5314  We use this feature heavily, and I'm the author of Ready Player Fun. This does not happen. What other mods do you have installed? I can figure it out, but there is something else going on, and I'll need a lot more information.
